core/node/crypto/chain_txpool.go

```diff
@@ -164,15 +164,18 @@ func newPendingTransactionPool(
 		"txpool_processed", "Number of submitted transactions that are included in the chain",
 		"chain_id", "address", "status",
 	)
+
+	// 1, 3, 5, 7, 9s
```

Comment:
@bas-vk does this look ok to you? This more than covers the range I've seen in omega over the past few days, with most tx inclusion <=5s and total inclusion <= 11s.